test/connection.spec.js

```diff
@@ -27,101 +24,89 @@ test('Fail because connection params are not there', () => {
 
 test('Fail because both host and address are missing', () => {
   expect(() => {
-    memgraph.Connect({ port: 7687, use_ssl: true });
+    memgraph.Connect({ port: 7687, use_ssl: false });
   }).toThrow();
 });
 
 test('Fail because port number is out of range', () => {
   expect(() => {
-    memgraph.Connect({ host: 'localhost', port: -100, use_ssl: true });
+    memgraph.Connect({ host: 'localhost', port: -100, use_ssl: false });
   }).toThrow();
   expect(() => {
-    memgraph.Connect({ host: 'localhost', port: 10000, use_ssl: true });
+    memgraph.Connect({ host: 'localhost', port: 10000, use_ssl: false });
   }).toThrow();
 });
 
 test('Connect to Memgraph host via SSL multiple times', async () => {
-  const port = await getPort();
-  await util.checkAgainstMemgraph(() => {
-    for (let iter = 0; iter < 100; iter++) {
-      const connection = memgraph.Connect({
-        address: '127.0.0.1',
-        port: port,
-        use_ssl: true,
-      });
-      expect(connection).toBeDefined();
-    }
-  }, port);
-});
-
-test('Connect to Memgraph address via SSL', async () => {
-  const port = await getPort();
-  await util.checkAgainstMemgraph(() => {
+  const port = 7687;
+  for (let iter = 0; iter < 100; iter++) {
     const connection = memgraph.Connect({
       address: '127.0.0.1',
       port: port,
-      use_ssl: true,
+      use_ssl: false,
     });
     expect(connection).toBeDefined();
-  }, port);
-});
-
-test('Fail because trust_callback is not callable', async () => {
-  const port = await getPort();
-  await util.checkAgainstMemgraph(() => {
-    expect(() => {
-      memgraph.Connect({
-        address: '127.0.0.1',
-        port: port,
-        use_ssl: true,
-        trust_callback: 'Not callable.',
-      });
-    }).toThrow();
-  }, port);
+  }
 });
 
-test('Fail because trust_callback returns false', async () => {
-  const port = await getPort();
-  await util.checkAgainstMemgraph(() => {
-    expect(() => {
-      memgraph.Connect({
-        address: '127.0.0.1',
-        port: port,
-        use_ssl: true,
-        trust_callback: () => {
-          return false;
-        },
-      });
-    }).toThrow();
-  }, port);
+test('Connect to Memgraph address via SSL', async () => {
+  const port = 7687;
+  const connection = memgraph.Connect({
+    address: '127.0.0.1',
+    port: port,
+    use_ssl: false,
```

Comment:
At least one test has to set `use_ssl: true`. It's a bit problematic within CI, one solution is to have 2 instances of Memgraph running. Another option is to do something similar to here https://github.com/memgraph/pymgclient/blob/master/test/common.py